lib/ts/recipe/thirdparty/types.ts

```diff
@@ -85,9 +83,10 @@ export type RecipeInterface = {
     }>;
 
     signInAndUp: (input: {
-        thirdPartyProviderId: string;
-        thirdPartyProviderClientId?: string;
-        thirdPartyRedirectionURL: string;
+        providerId: string;
+        redirectionURL: string;
+        providerClientId?: string;
+        authCode?: string;
```

Comment:
No need to give authCode

Reply:
- Actually we need to give it cause we had decided on the principle that tokens should be optionally inputted.
- Else, you can remove it, and then add a new recipe function to `getAuthCodeFromURL`. If this is being done, also change email verification recipe to add a `getEmailVerificationTokenFromURL`